config/locales/es/landing.es.yml

```diff
@@ -0,0 +1,3 @@
+es:
+  landing:
+    development: "Este sitio actualmente está en funcionamiento. Sin embargo, está en desarrollo. Puede que encuentre problemas técnicos (bugs). En ese caso, por favor cree un "issue" en %{github_link}, o envienos un %{contact_link} para poder resolverlo lo antes posible. Este proyecto es de código abierto. Te alentamos a contribuir en %{github_link}."
```

Comment:
I think the internal double quotes are throwing off the test. Those need to be escaped.

Reply:
I'm glad to know what was causing these errors.

BTW I don't think we use these strings anymore, so the file can just be deleted. I already deleted it in `locales/en`, so I think we probably want it deleted here, too.

Reply:
I think you can use single quotes (`'`) wrapping the outside of the whole translation, then use double quotes (`"`) within the translation string.